app/src/main/kotlin/com/metrolist/music/ui/component/SleepTimerDialog.kt

```
@@ -0,0 +1,439 @@
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SleepTimerTimePickerDialog(
+    title: String,
+    onDismiss: () -> Unit,
+    onConfirm: (String) -> Unit,
+    initialTime: String = DEFAULT_START
+) {
+    val timeFormatter    = DateTimeFormatter.ofPattern("HH:mm")
+    val initialLocalTime = try { LocalTime.parse(initialTime, timeFormatter) } catch (e: Exception) { LocalTime.of(9, 0) }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)